planet/clients/orders.py

```diff
@@ -344,27 +347,37 @@ async def poll(
         order_id: str,
         state: str = None,
         wait: int = 10,
-        verbose: bool = False
```

Comment:
Did you mean to get rid of this variable? The CLI current expects `verbose` as an input.